recipes/onedpl/all/conanfile.py

```diff
@@ -2,7 +2,7 @@
 
 from conans import ConanFile, CMake, tools
 
-required_conan_version = ">=1.28.0"
+required_conan_version = ">=1.36.0"
```

Comment:
```suggestion
required_conan_version = ">=1.43.0"
```
The failure happens because the targets are are absolute since 1.43.0 (second item in https://docs.conan.io/en/latest/changelog.html#dec-2021)